src/main/web_src/src/components/fagsystem/tpsf/form/personinformasjon/partials/Diverse.js

```diff
@@ -14,6 +16,15 @@ export const Diverse = ({ formikBag }) => {
 			formikBag.setFieldValue(`tpsf.bankkontonrRegdato`, null)
 		}
 	}
+	const opts = useContext(BestillingsveilederContext)
+	const { personFoerLeggTil } = opts
+	const harSkjerming = personFoerLeggTil
```

Comment:
Hva med 

```
const harSkjerming = personFoerLeggTil
   && personFoerLeggTil.tpsf.hasOwnProperty('egenAnsattDatoFom') 
   && personFoerLeggTil.tpsf.hasOwnProperty('egenAnsattDatoTom')
   && isAfter(new Date(personFoerLeggTil.tpsf.egenAnsattDatoTom), new Date())
```

Reply:
Ah, denne hadde jeg visst overtenkt skikkelig - bra innspill :)

Reply:
Nei vent, ser nå at det blir feil, for person som bare har egenAnsattDatoFom har skjerming. Setter den tilbake slik den var.